frontend/packages/local-storage-operator-plugin/src/components/local-volume-discovery/create-local-volume-discovery.tsx

```diff
@@ -49,11 +49,17 @@ export const CreateLocalVolumeDiscovery: React.FC<CreateLocalVolumeDiscoveryProp
 }) => {
   const { appName, ns } = match.params;
   const { t } = useTranslation();
+  const nameParamPresent = () => {
+    const params = new URLSearchParams(window.location.search.substring(1));
+    const nameParam = params.get('name');
+    return Boolean(nameParam);
+  };
 
+  const [discoveredNodesData] = useK8sWatchResource<NodeKind[]>(nodesDiscoveriesResource);
   const [nodesData, nodesLoaded, nodesLoadError] = useK8sWatchResource<NodeKind[]>(nodeResource);
   const [allNodes, setAllNodes] = React.useState([]);
   const [selectNodes, setSelectNodes] = React.useState([]);
```

Comment:
I think you can `setSelectNodes` only instead of adding one more prop with all selected nodes